pkg/runtime/path.go

```diff
@@ -0,0 +1,76 @@
+package runtime
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/alibaba/pouch/apis/types"
```

Comment:
I do not think a package in pkg should vendor package `github.com/alibaba/pouch/apis/types`. If we must do, I am afraid moving this package out of pkg is better.

Reply:
sure, if need.